### ISSUE TITLE
【Qusetion】about【power_to_db】

### ISSUE THREAD

**Issue by unbadfish** [librosa/librosa#1221]
Hi, Librosa Team:
I have two qusetions about the I/O "power_to_db"
When I use this I/O, it will return a 2d array. for further usage, I append each of these into a list. This is my code
```python
def anal_list(in_put):
    """[1, 2, 3]
    [4, 5, 6]
    [7, 8, 9]
    ->([[1, 2, 3], [4, 5, 6], [7, 8, 9]], 3)"""
    length_list: list = []
    output_list: list = []
    for x in in_put:
        output_list.append(list(x))
        length_list.append(x.shape[0])
    return output_list, max(length_list)
    # the list I am talking about is "output_list"
```
And I discover that the list has 1025 child list(index:0 - 1024)
====QUENSTION ONE====
The fig I got shows that the frequency "power_to_db"could analyse is in the range of 0-8192 and I guess each child list means a 'left-to-right' line. But I can't make sure what is the relationship between the Ordinate(Y-frequency) and the index of these child list?
What is the direction plt draw as the index increase?Does plt draw 'up-to down' or 'down-to-up'?
Or, in another word, let Y-frequency = f(index), please show me the f(x) or just show me the code that caculates the Y-frequency?
====QUENSTION TWO====
Each child list contains a lot of int-kind data. let's call their index in its list as 'index2'.
Plt would translate these data into a lot of dot, let's call the Abscissa of each dot as 'X-time'.
I am sure that the 'X-time' have some connection with 'X-time',  and also the sr of the audio I loaded.
Let X-time = g(index2, sr), please show me the expression of g(x, y), or show me the code that caculates g(x, y).
[Just use g(index2, sr), instead of g(x, y), to make it readable]
thank you very much ;)
Please forgive me for any possible grammar or spelling mistakes ~~and this silly qestion~~:smile:.

**Comment by bmcfee** [comment]
> The fig I got shows that the frequency "power_to_db"could analyse is in the range of 0-8192 and I guess each child list means a 'left-to-right' line. But I can't make sure what is the relationship between the Ordinate(Y-frequency) and the index of these child list?
> What is the direction plt draw as the index increase?Does plt draw 'up-to down' or 'down-to-up'?
> Or, in another word, let Y-frequency = f(index), please show me the f(x) or just show me the code that caculates the Y-frequency?

I don't think this has anything to do with `power_to_db`.  Power-to-db is an element-wise operation, so the output will have the same shape as the input, just with different values.

Otherwise, I'm not sure I understand your question without the code snippet you're describing.  Please fill out the issue template properly with a minimal example that reproduces the behavior you're asking about.
